2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -253,11 +261,17 @@ function mongo_reset_passwords() {
 
 # setup_keyfile fixes the bug in mounting the Kubernetes 'Secret' volume that
 # mounts the secret files with 'too open' permissions.
+# add --keyFile argument to mongo_common_args
 function setup_keyfile() {
+  # If user specify keyFile in cofig file do not use generated keyFile
+  if grep -q "keyFile" ${MONGODB_CONFIG_PATH}; then
+    exit 0
+  fi
   if [ -z "${MONGODB_KEYFILE_VALUE-}" ]; then
-    echo "ERROR: You have to provide the 'keyfile' value in $MONGODB_KEYFILE_VALUE"
+    echo "ERROR: You have to provide the 'keyfile' value in MONGODB_KEYFILE_VALUE"
```

Comment:
Don't know why this is still showing up here, but it is already in master... could you please rebase your PR (and probably squash the commits)?